index.html

```diff
@@ -391,11 +391,11 @@ <h4>Computation steps</h4>
                 <li id="comp_embedded_control_textbox"><em>Textbox:</em> If the embedded control has role <a class="role-reference" href="#textbox">textbox</a>, return its value.</li>
```

Comment:
This list is an "unorder list", and looks odd, given everything else is ordered now